browser/src/VariantList/sortUtilities.ts

```diff
@@ -1,30 +1,28 @@
 export const isEmpty = (val: any) => val === undefined || val === null || val === ''
```

Comment:
Since we're not altering this code apart from whitespace, let's drop this commit.